scripts/validate/shellcheck

```diff
@@ -1,11 +1,26 @@
 #!/usr/bin/env bash
-set -e
 
 # Maintain an array of files to shellcheck not the best solution but will do for the time being
 FILES=()
 FILES+=("contrib/completion/bash/docker")
+FILES+=("scripts/build/.variables")
+FILES+=("scripts/build/binary")
+FILES+=("scripts/build/cross")
+FILES+=("scripts/build/dynbinary")
+FILES+=("scripts/build/osx")
+FILES+=("scripts/build/windows")
+FILES+=("scripts/docs/generate-man.sh")
+FILES+=("scripts/docs/generate-yaml.sh")
+FILES+=("scripts/test/unit")
+FILES+=("scripts/test/unit-with-coverage")
+FILES+=("scripts/test/watch")
+FILES+=("scripts/validate/check-git-diff")
 FILES+=("scripts/validate/shellcheck")
 
+RETURN="0"
 for f in "${FILES[@]}"; do
     shellcheck "$f"
+    RETURN=$((RETURN+$?))
```

Comment:
I guess you could still support running them all and collecting the error codes, but I think the other changes would still be a good idea.